privacyidea/cli/privacyideatokenjanitor/findtokens.py

```diff
@@ -41,34 +43,31 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
+
+import click
+from flask.cli import AppGroup, FlaskGroup
+from click import ClickException
 from dateutil import parser
 from dateutil.tz import tzlocal, tzutc
-from flask_script.commands import InvalidCommand
 
 from privacyidea.lib.policy import ACTION
-from privacyidea.lib.utils import parse_legacy_time, to_unicode
+from privacyidea.lib.utils import parse_legacy_time
 from privacyidea.lib.importotp import export_pskc
 from privacyidea.lib.token import (get_tokens, remove_token, enable_token,
-                                   unassign_token, import_token,
+                                   unassign_token,
                                    get_tokens_paginated_generator)
 from privacyidea.models import Token
 from privacyidea.app import create_app
-from flask_script import Manager
 import re
 import sys
 from yaml import safe_dump as yaml_safe_dump
-from yaml import safe_load as yaml_safe_load
-
-
 
-__version__ = "0.1"
-
-ALLOWED_ACTIONS = ["disable", "delete", "unassign", "mark", "export", "listuser", "tokenrealms"]
+findtokens_cli = AppGroup("find", help="finds tokens and give the option to export, change or delete them.")
```

Comment:
We can drop the `AppGroup` here and just add the `find` command in `main.py` to the cli. This gets rid of the double use of `find`.